docs/what.rst

```diff
@@ -0,0 +1,23 @@
+=====================
+What is a Certificate?
+=====================
+
+An SSL *certificate* contains a public key and a private key that work together to enable
```

Comment:
Excellent feedback; very helpful. I'll take a look at this today or next week. What about the Lineages section?